packages/components/package.json

```
@@ -43,13 +43,13 @@
     "clean": "rimraf lib build",
     "cover": "c8 npm run -s test",
     "lint": "eslint ./src/**/*.{ts,tsx}",
-    "test:dev": "cross-env NODE_OPTIONS=\"--import presentation-test-utilities/node-hooks/ignore-styles\" mocha --enable-source-maps --config ./.mocharc.json",
+    "test:dev": "node --experimental-test-module-mocks --enable-source-maps --import presentation-test-utilities/node-hooks/ignore-styles ./node_modules/mocha/bin/mocha.js --config ./.mocharc.json",
```

Does it work like this?
```suggestion
    "test:dev": "cross-env NODE_OPTIONS=\"--experimental-test-module-mocks --enable-source-maps --import presentation-test-utilities/node-hooks/ignore-styles\" mocha --config ./.mocharc.json",
```

Would avoid pionting into `./node_modules` directly and leave it up to npm/pnpm to find mocha executable.

---

no, node complains that `--experimental-test-module-mocks` can't be passed through `NODE_OPTIONS` env var :(